l10n_eu_product_adr_local/models/adr_goods.py

```diff
@@ -0,0 +1,18 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import models, fields
+
```

Comment:
Between last import an Class definition OCA pre-commit will require two blank lines, so we should follow the same convention when possible
```suggestion


```

Reply:
This still missing. Two blank lines are required between imports and Classes, not between header comment and imports